### DISCUSSION TITLE
What is "Result class as IQuote"?  Can I contribute?

### DISCUSSION THREAD

**Discussion by NavjotSinghMinhas** [DaveSkender/Stock.Indicators#806] [Dev notes]
Hello,

I was thinking to contribute to this library but unfortunately, I am not sure where to start in the backlog queue like most of the items I am not sure of the strategy to follow or are not discussed in detail.

One item I am thinking to work on is "Result class as IQuote", can someone please explain it further, so I can contribute accordingly.

Thanks

**Comment by DaveSkender** [comment]
First off, thank you for your interest in contributing!  For this particular item, I probably should have just deleted it as it's going to be incorporated in #802 to enable #630 and other chaining approaches.  I've already started on these.

In the end, what I'm trying to do is make the results on one indicator reusable in another.  We've been able to do this simply for indicators that "look like" a quote, like Renko and Heikin-Ashi simply because they can follow the `IQuote` interface that is a common input for all indicators.

With that said, that doesn't work for any other indicators.  I have a plan to use a new interface that's more reusable, so I think I have that one figured out and have already started on it, but am happy to take input.

As someone new to the project, I'd suggest starting with the implementation of a regular indicator like #678 or any of the open candlestick patterns.

**Reply by DaveSkender** [reply]
If you’re going to pickup some work, just say so in the Issue.  If it’s just a note in the backlog, create an Issue for it to explain the problem and solution.

**Reply by DaveSkender** [reply]
Readability and chaining concepts were implemented in version 2.0
